generators/add-ci/.github/workflows/deploy_prod.yml

```diff
@@ -29,15 +29,26 @@ jobs:
       - name: Build
         env:
           AIO_RUNTIME_NAMESPACE: ${{ secrets.AIO_RUNTIME_NAMESPACE_PROD }}
-        uses: adobe/aio-apps-action@1.1.0
+        uses: adobe/aio-apps-action@2.0.0
```

Comment:
@sandeep-paliwal Could you also update the version in https://github.com/adobe/aio-apps-action/blob/master/package.json to 2.0.0 ?